docs/development_suite/governance/design-overview.md

```diff
@@ -89,7 +91,7 @@ The Microservices Overview collects data on the project configuration and create
 * *Memory Request*: Memory Request of the Microservice.
 * *Memory Limit*: Memory Limit of the Microservice.
 * *Log Parser*: Log Parser implementation of the Microservice.
-* *Advanced Configuration*: Advanced Configuration flag of the Microservice.
+* *Advanced Configuration*: Whether the Microservice is using raw manifest configuration.
```

Comment:
this part is no longer present in the general microservice overview

Reply:
no it refers to the design overview which maps the raw manifest conf mode to the column named advanced configuration